package-lock.json

```diff
@@ -3,16582 +3,6 @@
   "version": "0.0.0",
   "lockfileVersion": 2,
```

Comment:
@amdw  of course I agree, it has already been done.

it was just a doubt, i will use npm 7 too